src/js/selection/SelectionManager.js

```diff
@@ -91,7 +93,29 @@
     }
   };
 
-  ns.SelectionManager.prototype.paste = function() {
+  // Iterate through pixels to get the largest x and y values of a selection.
+  ns.SelectionManager.prototype._getBoundaryEdges = function(pixels) {
+    var xCoordinate = 0;
+    var yCoordinate = 0;
+    for (var i = 0; i < pixels.length; i++) {
+      if (xCoordinate < pixels[i].col) {
+        xCoordinate = pixels[i].col;
+      }
+      if (yCoordinate < pixels[i].row) {
+        yCoordinate = pixels[i].row;
```

Comment:
It might be easier to understand your goal if these were named `maxXCoordinate` and `maxYCoordinate`